src/i18n/pt-BR/index.js

```diff
@@ -0,0 +1,155 @@
+import potato from './potato';
+import print from './print';
+import tester from './tester';
+export default {
+  pt-BR: {
```

Comment:
@ThiagoWhispher this is the one that needs wrapping with single quotes.

Reply:
Done @yanfali :D